beacon_node/src/config.rs

```diff
@@ -433,7 +433,7 @@ pub fn get_config<E: EthSpec>(
             client_config
                 .network
                 .boot_nodes_enr
-                .extend_from_slice(boot_nodes)
+                .extend_from_slice(boot_nodes.as_slice())
```

Comment:
I don't think this is necessary? it's harmless anyway

Reply:
oh yeah nice catch. I had to add it in there to debug an issue with the mismatching ENR types. I'll remove it